### PR TITLE
consume new scroll state module with tests (bug 986625)

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

**Comment by ngokevin** [comment]
https://github.com/mozilla/marketplace-core-modules/pull/23/

File: tests/lib/helpers.js [hunk 1]
```diff
@@ -89,6 +89,16 @@ function waitForPageLoaded(cb) {
 }
 
 
+function waitForAppDetail(cb) {
+    casper.waitForSelector('[data-page-type~="detail"]', cb);
```
Comment (ngokevin) [inline tests/lib/helpers.js]: ur not defined

**Comment by chuckharmston** [comment]
Woof.

**Comment by muffinresearch** [comment]
Feels like we're being hounded. 

bad-um tish

**Pull request merged**